examples/bridge-app/esp32/main/main.cpp

```diff
@@ -230,7 +230,8 @@ EmberAfStatus HandleReadBridgedDeviceBasicAttribute(Device * dev, chip::Attribut
     }
     else if ((attributeId == ClusterRevision::Id) && (maxReadLength == 2))
     {
-        *buffer = (uint16_t) ZCL_BRIDGED_DEVICE_BASIC_INFORMATION_CLUSTER_REVISION;
+        uint16_t rev = ZCL_BRIDGED_DEVICE_BASIC_INFORMATION_CLUSTER_REVISION;
+        memcpy(buffer, &rev, sizeof(rev));
```

Comment:
Don't these need to be written in a specific byte order, via something like `chip::Encoding::LittleEndian::Put16()`

Reply:
No, the output of the external attribute callbacks is expected to be in native endianness for integer-typed attributes.